src/components/pill-selector-dropdown/__tests__/PillSelectorDropdown.test.js

```diff
@@ -73,6 +73,15 @@ describe('components/pill-selector-dropdown/PillSelectorDropdown', () => {
             instance.handleBlur();
             expect(addPillsFromInputMock).toHaveBeenCalledTimes(1);
         });
+
+        test.each([
+            ['test', true],
+            ['', false],
+        ])('label', (value, expected) => {
```

Comment:
can you make the test case description more specific? e.g. "should render Label component when label exists"

Reply:
Of course. Thanks for feedback